src/Classes/ServiceAPI/MyRadio_Show.php

```diff
@@ -1028,18 +1028,31 @@ public static function getMostListened($date = 0)
 
     /**
      * Searches searchable *text* metadata for the specified value. Does not work for image metadata.
+     * if $q is set, then $path_query must be set to "search"
      *
      * @todo effective_from/to not yet implemented
      *
-     * @param string $query          The query value.
+     * @param string $path_query     The query value encoded in the path (DEPRECATED).
+     * @param string $q              The query value as a query string. to use this, $path_query must be set to "search"
```

Comment:
you've included this comment twice however you haven't explained why this behaviour exists

Reply:
how parameters work in the serviceapi is weird. its a bit clearer in the autogenerated api docs but i will add more details to it